### PR TITLE
Basic HTTP Backendserver to check merkle airdrop entitlement

### PR DESCRIPTION
**Pull request by jondoe1337**
Fixes server part of https://github.com/trustlines-network/project/issues/352

### CODE REVIEW

File: merkle_drop/server.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+def bad_request(e):
+    return jsonify(error=400, message=e.description), 400
+
+
+@app.errorhandler(500)
+def internal_server_error(e):
+    return jsonify(error=500, message="There was an internal server error"), 500
+
+
+@app.route("/airdrop/<string:address>", methods=["GET"])
```
Comment (berndbohmeier) [inline merkle_drop/server.py]: It is not really clear what will be returned. 
Maybe `/entitlement/addresses/<string:address>`?

File: merkle_drop/server.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+    eligible_tokens = get_balance(canonical_address, airdrop_dict)
+    if eligible_tokens == 0:
+        abort(404)
+    proof = create_proof(get_item(canonical_address, airdrop_dict), airdrop_tree)
+    # TODO: apply decay function on tokens
+    return jsonify(
+        {
+            "address": address,
+            "eligible_token_balance": eligible_tokens,
```
Comment (berndbohmeier) [inline merkle_drop/server.py]: use camelCase for json.
What about `originalTokenBalance`, because we later will need `currentTokenBalance`?

**Comment by berndbohmeier** [comment]
And ofc please fix black and tests.

**Comment by jondoe1337** [comment]
> And ofc please fix black and tests.

btw: I would recommend putting essential tasks that should be done in pre-commit hooks.

File: merkle_drop/server.py (hunk 2 above)
Comment (jondoe1337) [reply merkle_drop/server.py]: `originalTokenBalance` could be added, but where would the `currentTokenBalance` come from?

File: merkle_drop/server.py (hunk 1 above)
Comment (jondoe1337) [reply merkle_drop/server.py]: Okay `airdrop` might be some kind of meta-word. I'm happier with `entitlement`, but not with added `addresses` as it is not RESTful.

Comment (berndbohmeier) [reply merkle_drop/server.py]: Okay, I am also fine without `addresses`

File: merkle_drop/server.py (hunk 2 above)
Comment (berndbohmeier) [reply merkle_drop/server.py]: I would just rename eligible_token_balance to `originalTokenBalance`. `currentTokenBalance` will then later(not in this pr) be the value with the decay.

**Pull request merged**